README.md

```diff
@@ -474,6 +474,15 @@ Preview classes live in `test/components/previews`, which can be configured usin
 config.view_component.preview_path = "#{Rails.root}/lib/component_previews"
 ```
 
+Previews are served from <http://localhost:3000/rails/view_components> by default. If you want to use a different endpoint you can configure the `preview_route` option:
+
+`config/application.rb`
+```ruby
+config.view_component.preview_route = "/previews"
```

Comment:
I think we'd call this `preview_path` ?

Reply:
We already have that config for enabling users configure where the previews live in the codebase. Should we rename?

Reply:
oh right. hmm... I'd say `preview_source_path` and `preview_path` but that's a confusing breaking change. Let's debate the mount idea before we are forced to decide on this.

Reply:
I prefer this approach to the mounted engine implementation. I also think `preview_route` is fine 👍